debian/libibverbs1.symbols

```diff
@@ -1,6 +1,8 @@
 libibverbs.so.1 libibverbs1 #MINVER#
  IBVERBS_1.0@IBVERBS_1.0 1.1.6
  IBVERBS_1.1@IBVERBS_1.1 1.1.6
+ IBVERBS_1.3@IBVERBS_1.3 1.1.12
+ IBVERBS_1.4@IBVERBS_1.4 12
```

Comment:
@talatb I checked it and my pull request has the same patches as you sent them to me.

I'll send two pull request now:
one with approved patches by @bdrung 
* debian: Add lsb-base dependency for ibacm, iwpmd and srptools
* debian: Add source function library for srptools daemon
* debian: Switch to dpkg-source 3.0 (quilt) format

and the second with the rest patches